csharp/client/Dhe_NetClient/session/DndTableHandleManager.cs

```diff
@@ -0,0 +1,45 @@
+﻿//
+// Copyright (c) 2016-2025 Deephaven Data Labs and Patent Pending
+//
+using Deephaven.Dh_NetClient;
+using Io.Deephaven.Proto.Backplane.Grpc;
+
+namespace Deephaven.Dhe_NetClient;
+
+public class DndTableHandleManager : TableHandleManager {
+  internal DndTableHandleManager(Ticket? consoleId, Server server) : base(consoleId, server) {
+  }
+
+  /// <summary>
+  /// Fetches a historical table from the database on the server.
+  /// </summary>
+  /// <param name="tableNamespace">The namespace of the table to fetch</param>
+  /// <param name="tableName">the name of the table to fetch</param>
+  /// <returns>the TableHandle of the fetched table.</returns>
+  public TableHandle HistoricalTable(string tableNamespace, string tableName) {
+    throw new NotImplementedException();

```

Comment:
Is this planned work to implement? Or is there some reason this is impossible?

Reply:
Fixed, thank you